kv/bucket.go

```diff
@@ -327,24 +323,45 @@ func (s *Service) FindBuckets(ctx context.Context, filter influxdb.BucketFilter,
 		if err != nil {
 			return err
 		}
+
 		bs = bkts
+
 		return nil
 	})
 
-	if err != nil {
-		return nil, 0, err
+	needsSystemBuckets := true
+	for _, b := range bs {
+		if b.Type == influxdb.BucketTypeSystem {
+			needsSystemBuckets = false
+			break
+		}
 	}
 
-	tasks, error := s.findSystemBucket("_tasks")
-	if error != nil {
-		return bs, 0, error
+	if needsSystemBuckets {
+		tb := &influxdb.Bucket{
+			ID:              influxdb.TasksSystemBucketID,
+			Type:            influxdb.BucketTypeSystem,
+			Name:            influxdb.TasksSystemBucketName,
+			RetentionPeriod: time.Hour * 24 * 3,
+			Description:     "System bucket for task logs",
+		}
+
+		bs = append(bs, tb)
+
+		mb := &influxdb.Bucket{
+			ID:              influxdb.MonitoringSystemBucketID,
+			Type:            influxdb.BucketTypeSystem,
+			Name:            influxdb.MonitoringSystemBucketName,
+			RetentionPeriod: time.Hour * 24 * 7,
+			Description:     "System bucket for monitoring logs",
+		}
+
+		bs = append(bs, mb)
 	}
 
-	monitoring, error := s.findSystemBucket("_monitoring")
-	if error != nil {
-		return bs, 0, error
+	if err != nil {
+		return nil, 0, err
 	}
-	bs = append(bs, tasks, monitoring)
 
 	return bs, len(bs), nil
```

Comment:
This is not related to your changes but.. I have no idea why we need to return the length of a slice along with the actual slice...